packages/FormElement/src/FormElement.js

```diff
@@ -0,0 +1,169 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import isNil from "lodash/isNil";
+import isString from "lodash/isString";
+import uuid from "uuid/v1";
+
+import { extractChildren } from "./helpers/extractChildren";
+import Description from "./components/Description";
+import ErrorMessage from "./components/ErrorMessage";
+import Hint from "./components/Hint";
+import Label from "./components/Label";
+
+import formElementStyles from "./FormElement.styles";
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+
+  /** ID for the child element. */
+  id: PropTypes.string,
+
+  /** Should be disabled or not, default is false. */
+  isDisabled: PropTypes.bool,
+
+  /** Should label and children be inline or not, default is false. */
+  isInline: PropTypes.bool,
+
+  /** Should label be hidden, default is false. Note: this is discouraged because of accessibility requirements. */
+  isLabelVisuallyHidden: PropTypes.bool,
+
+  /** Should show is optional text besides the label or not. */
+  isOptional: PropTypes.bool,
+
+  /** Should be read-only or not, default is false. */
+  isReadOnly: PropTypes.bool,
+
+  /** Should show is required text besides the label or not. */
+  isRequired: PropTypes.bool,
```

Comment:
To me this feels or can make the impression that the `<FormElement />` will be responsible for validating the field which is not the case. 

I might suggest the use of another prop like kind="optional" || kind="required" instead of isOptional or isRequired

Reply:
Decided use `hasOptionalLabel` `hasRequiredLabel` instead.